troubleshooting/tools/tracevol.py

```diff
@@ -58,16 +61,24 @@
                     default="admin", help="user ID to connect to ceph cluster")
 PARSER.add_argument("-key", "--userkey",
                     default="", help="user password to connect to ceph cluster")
+PARSER.add_argument("-cm", "--configmap", default="ceph-csi-config",
+                    help="config map where subvolumegroup is specified")
 
 
 def list_pvc_vol_name_mapping(arg):
     """
     list pvc and volume name mapping
     """
-    table = prettytable.PrettyTable(
-        ["PVC Name", "PV Name", "Image Name", "PV name in omap",
-         "Image ID in omap", "Image in cluster"]
-    )
+    table_rbd = prettytable.PrettyTable()
+    table_rbd.title = "RBD"
+    table_rbd.field_names = ["PVC Name", "PV Name", "Image Name", "PV name in omap",
+                             "Image ID in omap", "Image in cluster"]
+
+    table_cephfs = prettytable.PrettyTable()
+    table_cephfs.title = "CephFS"
+    table_cephfs.field_names = ["PVC Name", "PV Name", "Subvolume Name", "PV name in omap",
+                                "Subvolume ID in omap", "Subvolume in cluster"]
```

Comment:
are we storing `Subvolume ID` in omap :thinking: ?

Reply:
Yes, check_image_uuid_in_rados() checks the same. image_id for CephFS here means subvolume id.

Reply:
its basically as subvolume name and for rbd its image name, we can correct it later